CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+# 2.7.7
```

Comment:
Since this adds a new feature, it needs to increment the minor version, not the patch version (second number, not third). See also https://semver.org/